### ISSUE TITLE
Audio files upload support

### ISSUE THREAD

**Issue by shpingalet007** [pocketnetteam/pocketnet.gui#628]
## What need to be done
We could provide support for audio media files. This will allow to load music, recordings, etc.

## Details
After digging around Peertube, looks like it converts audio formats to video. The best we can do with it is create a video file on a backend that will be provided with minimal video stream and minimal bit rate.

I removed the restriction of audio upload from frontend and [created a video](https://peertube21.pocketnet.app/w/nMd4kE7oPRdawKSLy7RF9K) based on my audio (empty by the way, sorry no mic).

This video has several resolutions... WAT?! Okay, we need just determine that it is audio and keep only one unique resolution e.g. ~10x10 pixels~ (checked, it can have 0x0 without any problems) just to be able to use the MP4 file without breaking the Peertube ecosystem.

## TLDR;
- **Backend**
  - Keep only 1 unique resolution for audio
  - Reduce frame rate for the created video
  - Create some sort of flag to differentiate between video and audio `isAudio: true` and `isVideo: true`
- **Frontend**
  - Create audio player UI
  - Create floating mode (some sort of picture-in-picture mode, but for audio media)
  - Other UI corrections. There are many localizations, categories, windows speaking only about video

## Involved devs
- @Traler will do the code
- @zanhesl will be helpful by providing Peertube related information
- @shpingalet007 will snoop around when it needed to provide any possible help as senior

**Comment by Traler** [comment]
I will do

**Comment by shpingalet007** [comment]
This task is still not done, could be assigned to some dev

**Comment by shpingalet007** [comment]
Completed in 2023
